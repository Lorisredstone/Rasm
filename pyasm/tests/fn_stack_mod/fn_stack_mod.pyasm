include std/std.pyasm

func print_42
    42
end

print_42 .