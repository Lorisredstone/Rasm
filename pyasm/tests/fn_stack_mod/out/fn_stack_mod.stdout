42
5000
42
